codegen/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojo.java

```diff
@@ -111,8 +114,11 @@ public void setIosCustomElements(String[] elements) {
     public void setAndroidCustomElements(String[] elements) {
         androidCustomElements = Arrays.asList(elements);
     }
-    
-    
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2360069942/issues/source?bid=2928425&fileBranchId=3219780#l118)